editor/components/post-featured-image/index.js

```diff
@@ -22,9 +23,26 @@ import MediaUpload from '../media-upload';
 const DEFAULT_SET_FEATURE_IMAGE_LABEL = __( 'Set featured image' );
 const DEFAULT_REMOVE_FEATURE_IMAGE_LABEL = __( 'Remove image' );
 
-function PostFeaturedImage( { featuredImageId, onUpdateImage, onRemoveImage, media, postType } ) {
+function PostFeaturedImage( { currentPostId, featuredImageId, onUpdateImage, onRemoveImage, media, postType } ) {
 	const postLabel = get( postType, [ 'labels' ], {} );
 
+	let mediaWidth,
+		mediaHeight,
```

Comment:
What's up with the indentation here? If these have to be on their own lines it might just be nicer to have separate `let` declarations. 🤷‍♂️

Reply:
> If these have to be on their own lines it might just be nicer to have separate `let` declarations

Is there an eslint for this coding standard?

Reply:
I don't think so, but there aren't any instances of:

```js
let foo,
    bar,
    foobar;
```

in the code when I looked for it. Conversely, there are several instances of multiple-variable `let` declarations:

https://github.com/WordPress/gutenberg/blob/master/packages/date/src/index.js#L285
https://github.com/WordPress/gutenberg/blob/master/packages/components/src/toggle-control/index.js#L35
https://github.com/WordPress/gutenberg/blob/master/packages/components/src/higher-order/with-focus-outside/test/index.js#L17

Reply:
My implicit point was: if it's coding style and there's an eslint for it, then it's enforceable. If there isn't a lint for it, then it's a bit perfectionist and detracts from the code review process.

Regardless, I've updated in da643a145